options/locale/locale_en-US.ini

```diff
@@ -1167,6 +1167,8 @@ branch.protected_deletion_failed = Branch '%s' is protected. It cannot be delete
 
 topic.manage_topics = Manage Topics
 topic.done = Done
+topic.count_prompt = You can't select more than 25 topics
+topic.format_prompt = Topics must use letter or number and can include hyphen(-), underscore(_), plus(+), hash(#), dot(.) with max length of 35
```

Comment:
And error message if changed to the regexp above would be: `Topics must start with letter or number, can include hyphen(-) must be no more than 35 characters long` or something like that :)

Reply:
Done